<?xml version="1.0" encoding="utf-8"?>
<log>
<entry code="info" level="info">
 <message>[info] Activation of plug-in DQC Rules Validator successful, version Check version using Tools->DQC->Version on the GUI or --dqc-version on the command line. - validate/DQC </message>
 <ref href="validate/DQC"/></entry>
<entry code="info" level="info">
 <message>[info] Activation of plug-in SEC Inline Transforms successful, version 19.2. - transforms/SEC </message>
 <ref href="transforms/SEC"/></entry>
<entry code="info" level="info">
 <message>[info] Xule version: 3.0.23379 - </message>
 <ref href=""/></entry>
<entry code="xmlSchema:syntax" level="error">
 <message fileName="jwn-20220129.htm" error="Specification mandates value for attribute defer" line="4" column="159">[xmlSchema:syntax] Specification mandates value for attribute defer, jwn-20220129.htm, line 4, column 159 - ../../../../../../../jwn-20220129.htm , http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm </message>
 <ref href="http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm"/>
 
 <ref href="../../../../../../../jwn-20220129.htm"/></entry>
<entry code="html:syntaxError" level="error">
 <message element="Html" error="Element script does not carry attribute type line 4, at path /html/head/script[1], Element noscript content does not follow the ...">[html:syntaxError] Html error Element script does not carry attribute type line 4, at path /html/head/script[1], Element noscript content does not follow the DTD, expecting (p | h1 | h2 | h3 | h4 | h5 | h6 | div | ul | ol | dl | pre | hr | blockquote | address | fieldset | table | form | noscript | ins | del | script | ix:footnote | ix:nonNumeric | ix:fraction | ix:numerator | ix:denominator | ix:nonFraction | ix:continuation | ix:exclude)*, got (img) line 4, at path /html/body/noscript, Element img does not carry attribute alt line 4, at path /html/body/noscript/img - http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm 4</message>
 <ref href="http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm#element(/1/1/3)" sourceLine="4">
  <property name="QName" value="script"/>
 </ref>
 
 <ref href="http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm#element(/1/2/924)" sourceLine="4">
  <property name="QName" value="noscript"/>
 </ref>
 
 <ref href="http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm#element(/1/2/924/1)" sourceLine="4">
  <property name="QName" value="img"/>
  <property name="src" value="https://www.sec.gov/akam/13/pixel_7ba342d9?a=dD1hODQxZTI3NmU4MTg1NmUyZjQwNWJmZDJlMmMyNGFjYjczNTE2MjM3JmpzPW9mZg=="/>
  <property name="style" value="visibility: hidden; position: absolute; left: -999px; top: -999px;"/>
 </ref></entry>
<entry code="info" level="info">
 <message>[info] loaded in 6.65 secs at 2022-11-04T15:05:22 - http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm </message>
 <ref href="http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm"/></entry>
<entry code="info" level="info">
 <message>[info] Activation of package dqc_15_concepts.csv, dqc_0011.csv, dqc_0079.csv, dqc_0080.csv, dqc_0081.csv, dqc_0091_ifrs_2018_concepts.csv, dqc_0091_ifrs_2019_concepts.csv, dqc_0094.csv, dqc_0129.csv and effective_dates.csv successful. - D:\Temp\tmp0b7o2s9f\packages\resources.zip </message>
 <ref href="D:\Temp\tmp0b7o2s9f\packages\resources.zip"/></entry>
<entry code="MemoryError" level="error">
 <message fileName="us-gaap-2021-01-31.xsd" error="">[MemoryError] Unrecoverable error: , us-gaap-2021-01-31.xsd - https://xbrl.fasb.org/us-gaap/2021/elts/us-gaap-depcon-pre-2021-01-31.xml 10
Traceback (most recent call last):
  File &quot;Z:\Users\arelle\devpub\arelle\ModelDocument.py&quot;, line 156, in load
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 506, in file
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 693, in openXmlFileStream
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 68, in __init__
MemoryError</message>
 <ref href="https://xbrl.fasb.org/us-gaap/2021/elts/us-gaap-depcon-pre-2021-01-31.xml#element(/1/2/1)" sourceLine="10"/></entry>
<entry code="MemoryError" level="error">
 <message fileName="us-gaap-lab-2021-01-31.xml" error="">[MemoryError] Unrecoverable error: , us-gaap-lab-2021-01-31.xml - https://xbrl.fasb.org/us-gaap/2021/elts/us-gaap-std-2021-01-31.xsd 10
Traceback (most recent call last):
  File &quot;Z:\Users\arelle\devpub\arelle\ModelDocument.py&quot;, line 156, in load
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 506, in file
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 680, in openXmlFileStream
MemoryError</message>
 <ref href="https://xbrl.fasb.org/us-gaap/2021/elts/us-gaap-std-2021-01-31.xsd#element(/1/1/1/1)" sourceLine="10">
  <property name="QName" value="link:linkbaseRef"/>
  <property name="xlink:arcrole" value="http://www.w3.org/1999/xlink/properties/linkbase"/>
  <property name="xlink:href" value="us-gaap-lab-2021-01-31.xml"/>
  <property name="xlink:role" value="http://www.xbrl.org/2003/role/labelLinkbaseRef"/>
  <property name="xlink:type" value="simple"/>
 </ref></entry>
<entry code="MemoryError" level="error">
 <message fileName="us-gaap-stm-scf-re-def-2021-01-31.xml" error="">[MemoryError] Unrecoverable error: , us-gaap-stm-scf-re-def-2021-01-31.xml - https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-scf-re-2021-01-31.xsd 11
Traceback (most recent call last):
  File &quot;Z:\Users\arelle\devpub\arelle\ModelDocument.py&quot;, line 156, in load
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 506, in file
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 693, in openXmlFileStream
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 68, in __init__
MemoryError</message>
 <ref href="https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-scf-re-2021-01-31.xsd#element(/1/1/1/2)" sourceLine="11">
  <property name="QName" value="link:linkbaseRef"/>
  <property name="xlink:arcrole" value="http://www.w3.org/1999/xlink/properties/linkbase"/>
  <property name="xlink:href" value="us-gaap-stm-scf-re-def-2021-01-31.xml"/>
  <property name="xlink:role" value="http://www.xbrl.org/2003/role/definitionLinkbaseRef"/>
  <property name="xlink:type" value="simple"/>
 </ref></entry>
<entry code="MemoryError" level="error">
 <message fileName="us-gaap-stm-soi-pre-2021-01-31.xml" error="">[MemoryError] Unrecoverable error: , us-gaap-stm-soi-pre-2021-01-31.xml - https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-2021-01-31.xsd 11
Traceback (most recent call last):
  File &quot;Z:\Users\arelle\devpub\arelle\ModelDocument.py&quot;, line 156, in load
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 506, in file
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 693, in openXmlFileStream
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 68, in __init__
MemoryError</message>
 <ref href="https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-2021-01-31.xsd#element(/1/1/1/2)" sourceLine="11">
  <property name="QName" value="link:linkbaseRef"/>
  <property name="xlink:arcrole" value="http://www.w3.org/1999/xlink/properties/linkbase"/>
  <property name="xlink:href" value="us-gaap-stm-soi-pre-2021-01-31.xml"/>
  <property name="xlink:role" value="http://www.xbrl.org/2003/role/presentationLinkbaseRef"/>
  <property name="xlink:type" value="simple"/>
 </ref></entry>
<entry code="MemoryError" level="error">
 <message fileName="us-gaap-stm-soi-cal-2021-01-31.xml" error="">[MemoryError] Unrecoverable error: , us-gaap-stm-soi-cal-2021-01-31.xml - https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-2021-01-31.xsd 12
Traceback (most recent call last):
  File &quot;Z:\Users\arelle\devpub\arelle\ModelDocument.py&quot;, line 156, in load
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 506, in file
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 693, in openXmlFileStream
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 68, in __init__
MemoryError</message>
 <ref href="https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-2021-01-31.xsd#element(/1/1/1/3)" sourceLine="12">
  <property name="QName" value="link:linkbaseRef"/>
  <property name="xlink:arcrole" value="http://www.w3.org/1999/xlink/properties/linkbase"/>
  <property name="xlink:href" value="us-gaap-stm-soi-cal-2021-01-31.xml"/>
  <property name="xlink:role" value="http://www.xbrl.org/2003/role/calculationLinkbaseRef"/>
  <property name="xlink:type" value="simple"/>
 </ref></entry>
<entry code="MemoryError" level="error">
 <message fileName="us-gaap-stm-soi4-cal-2021-01-31.xml" error="">[MemoryError] Unrecoverable error: , us-gaap-stm-soi4-cal-2021-01-31.xml - https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-2021-01-31.xsd 13
Traceback (most recent call last):
  File &quot;Z:\Users\arelle\devpub\arelle\ModelDocument.py&quot;, line 156, in load
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 506, in file
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 693, in openXmlFileStream
  File &quot;Z:\Users\arelle\devpub\arelle\FileSource.py&quot;, line 68, in __init__
MemoryError</message>
 <ref href="https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-2021-01-31.xsd#element(/1/1/1/4)" sourceLine="13">
  <property name="QName" value="link:linkbaseRef"/>
  <property name="xlink:arcrole" value="http://www.w3.org/1999/xlink/properties/linkbase"/>
  <property name="xlink:href" value="us-gaap-stm-soi4-cal-2021-01-31.xml"/>
  <property name="xlink:role" value="http://www.xbrl.org/2003/role/calculationLinkbaseRef"/>
  <property name="xlink:type" value="simple"/>
 </ref></entry>
<entry code="webCache:retryingOperation" level="error">
 <message error="" URL="https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-def-2021-01-31.xml" retryCount="5">[webCache:retryingOperation]  
unsuccessful retrieval of https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-def-2021-01-31.xml 
5 retries remaining - </message>
 <ref href=""/></entry>
<entry code="webCache:retryingOperation" level="error">
 <message error="" URL="https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-def-2021-01-31.xml" retryCount="4">[webCache:retryingOperation]  
unsuccessful retrieval of https://xbrl.fasb.org/us-gaap/2021/stm/us-gaap-stm-soi-def-2021-01-31.xml 
4 retries remaining - </message>
 <ref href=""/></entry>
<entry code="info" level="info">
 <message>[info] validated in 92.69 secs - http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm </message>
 <ref href="http://www.sec.gov/Archives/edgar/data/72333/000007233322000060/jwn-20220129.htm"/></entry>
</log>
